tests/drive_failure/drive_failure_test.go

```diff
@@ -52,13 +52,15 @@ var _ = Describe("{DriveFailure}", func() {
 				})
 
 				driveInfoMap := make(map[string]string)
+				busInfoMap := make(map[string]string)
 				Step(fmt.Sprintf("induce a failure on all drives on the node %v", nodeWithDrive), func() {
 					for _, driveToFail := range drives {
-						driveID, err := Inst().N.YankDrive(nodeWithDrive, driveToFail, node.ConnectionOpts{
+						driveID, busID, err := Inst().N.YankDrive(nodeWithDrive, driveToFail, node.ConnectionOpts{
 							Timeout:         1 * time.Minute,
 							TimeBeforeRetry: 5 * time.Second,
 						})
 						driveInfoMap[driveToFail] = driveID
```

Comment:
If driveInfoMap is not required anymore, we can get rid of it.
YankDrive can just return the BusID if that is more reliable for the RecoverDrive command.